keylime/tpm2.py

```diff
@@ -402,8 +428,10 @@ def __create_ek(self, asym_alg=None):
             if code != tpm_abstract.AbstractTPM.EXIT_SUCESS:
                 raise Exception("createek failed with code "+str(code)+": "+str(output))
 
-            if legacy_tools:
+            if tools_version == "3.2":
                 handle = int(0x81010007)
+            elif tools_version == "4.0":
+                handle = int(0x81010009)
```

Comment:
Get rid of `-Q` above and change back to `-c -` and this can be removed for `4.0` tools.